lib/output/ebook/getConvertOptions.js

```diff
@@ -65,6 +65,10 @@ function getConvertOptions(output) {
                 '--pdf-header-template':    headerTpl,
                 '--pdf-footer-template':    footerTpl
             });
+            if (pdfOptions.customSize) {
+                options['--custom-size'] = String(pdfOptions.customSize);
```

Comment:
All this does is allows --custom-size to pass through to calibre.  You can then add what size you want to your book.json file.

eg:

```
{
  "pdf": {
    "customSize": "8.12x10.25"
  }
}
```

Then when you run 'gitbook pdf' it will create the size you want.

If you are on OSX, you can just patch your currently installed version.  Replace the version number with whatever you are running.

```
vi ~/.gitbook/versions/3.2.2/lib/output/ebook/getConvertOptions.js
```